types/dygraphs/index.d.ts

```diff
@@ -1284,6 +1284,16 @@ declare class Dygraph {
      */
     indexFromSetName(name: string): number;
 
+    /**
+     * Find the row number corresponding to the given x-value.
+     * Returns null if there is no such x-value in the data.
+     * If there are multiple rows with the same x-value, this will return the
+     * first one.
+     * @param {number} xVal The x-value to look for (e.g. millis since epoch).
+     * @return {?number} The row number, which you can pass to getValue(), or null.
```

Comment:
Remove the type annotations in the JSDoc. At best they are duplicative and at worst they disagree (as in this case).

Reply:
Thanks, done.